### DISCUSSION TITLE
Troubleshooting

### DISCUSSION THREAD

**Discussion by giscus** [arcjet/arcjet-docs#312] [General]
# Troubleshooting

Debugging common problems and errors with Arcjet.

https://docs.arcjet.com/troubleshooting/

**Comment by Vionsys-Official** [comment]
my whole application uses import and export without any error but when i tried to use arcjet it shows error of The current file is a CommonJS module whose imports will produce ‘require’ calls !!

**Reply by davidmytton** [reply]
Hi @Vionsys-Official ! Arcjet is ESM only so if you're seeing that error then your app is probably configured as CommonJS. Can you check the troubleshooting guide at https://docs.arcjet.com/troubleshooting#the-current-file-is-a-commonjs-module-whose-imports-will-produce-require-calls to see if the suggestions there work?

If not, can you let me know what you have in your `package.json`?
